addons/mail/static/tests/messaging_menu/messaging_menu.test.js

```diff
@@ -1220,33 +1221,33 @@ test("can open messaging menu even if messaging is not initialized", async () =>
     patchBrowserNotification("default");
     await startServer();
     const def = new Deferred();
-    onRpcBefore("/web/dataset/call_kw/ir.http/lazy_session_info", async () => {
+    const { waitStoreFetch } = listenStoreFetch("init_messaging", async () => {
+        asyncStep("before init_messaging");
```

Comment:
Note: this step is necessary to ensure the callback is actually applied, it makes the test much more robust.